packages/docusaurus-theme-classic/src/theme/BlogPostAuthor/index.tsx

```diff
@@ -24,11 +22,11 @@ export default function BlogPostAuthor({author}: Props): JSX.Element {
   return (
     <div className="avatar margin-bottom--sm">
       {imageURL && (
-        <span className="avatar__photo-link avatar__photo">
-          <MaybeLink href={link}>
-            <img className={styles.image} src={imageURL} alt={name} />
-          </MaybeLink>
-        </span>
+        <MaybeLink
+          href={link}
+          className={link ? 'avatar__photo-link' : undefined}>
```

Comment:
The `MaybeLink` will not apply other props if `href` is falsy. You can remove this check.